guardian/content-api-models
# Remove duplicated tests that are no longer necessary

## What does this change?

Support for Scala 2.11 was removed here: https://github.com/guardian/content-api-models/commit/ab78878c0ebee5dbfbd980c28ba6d44fe0980ce8. ~This pr adds tests for Scala 2.13 and removes 2.11 tests.~

EDIT: This PR now removes the 2.11 tests entirely, and makes the existing 2.12 tests run for all supported scala versions. This means the repository tests are now run for scala 2.13 as well as for scala 2.12. See messages below for more detail.

## How to test

- Compare `sbt "clean; test"` on main with this branch: see that the former compiles source code but doesn’t run any tests, whereas the latter compiles code and runs tests
- Compare `sbt "clean; +test"` on main with this branch: on main the tests are only run for scala 2.12, but on this branch they’re also run for 2.13.
